script/Ciel/Day 13/13_01_01_CIEL13_3_1.txt

```diff
@@ -867,7 +928,7 @@ It pierced straight through my chest, but the wound has already stopped bleeding
 [sha:596f804abe87c9b9ad267c5dd7d16c8524c6c9d5]{
 -- Page 302, Offset 34071.
 -- 俺はもう、完全に人間ではなくなっている―――
-I've already lost the humanity that I once had―――
+At this point, I'm no longer even human at all―――
 }
 [sha:ef05206d91dd0baec21c216d2852863122fd144b]{
 -- Page 303, Offset 34072.
```

Comment:
This a bit of a JPism (Offset 34072). Maybe something like this.
>My consciousness blanks over. 

Offset 34073-75 should probably follow the same pattern.
>Don't let her deceive you.

>Don't let her deceive you.

>Hasn't she been deceiving you this entire entire time?